config/django_config/production.py

```
@@ -7,11 +7,13 @@
 from .base import *  # noqa
 
 DEBUG = env.bool("DJANGO_DEBUG", default=False)
```

Ruff (0.8.2)

`env` may be undefined, or defined from star imports

(F405)